spec/features/create_and_deliver_a_weekly_digest_spec.rb

```diff
@@ -0,0 +1,283 @@
+RSpec.describe "create and delive a weekly digest", type: :request do
+  include UTMHelpers
+
+  before do
+    allow_any_instance_of(DeliveryRequestService)
+      .to receive(:provider_name).and_return("notify")
+
+    stub_notify
+  end
+
+  scenario do
+    login_with_internal_app
+
+    # create two subscriber lists with different links
+    list_one_topic_id = "0eb5d0f0-d384-4f27-9da8-3f9e9b22a820"
+    list_one_taxon_id = "86db0cbd-a1f9-4218-b571-ca0550265e33"
```

Comment:
```suggestion
    list_two_taxon_id = "86db0cbd-a1f9-4218-b571-ca0550265e33"
```

This appears to only be used for the second list.

Reply:
Oh good spot.